Feature/i18n content

DSVDavid opened the pull request:
<!--
### Requirements for making a pull request

Thank you for contributing to our project!

Please fill out the template below to help the project maintainers review it as fast as possible and include your contribution to the project.
-->

### What does it fix?
Adds enpoint for admin to create/update string resource and enpoints for i18n content related data

Closes #49  

- [x] renamed `LocalizationController` to `AdminLocalizationController`
- [x] changed route to `admin/content`
- [x] created `AddOrUpdateAsync` method
- [x] added project called `DeUrgenta.Contnent.Api` 
- [x] added `ContentController`
- [x] added method `GetAvailableContent(string key)`
- [x] added method `GetAvailableContentKeys()`
- [x] added method `GetAvailableLanguages()` 


### How has it been tested?
Tested using the Swagger UI

DSVDavid committed:
-created new project DeUrgenta.Content.Api
-added get languages method
-added get keys for header language method
-create/update language key method

DSVDavid committed:
-small changes

DSVDavid committed:
Merge branch 'code4romania:develop' into feature/i18n_content

idormenco commented on Src/DeUrgenta.Admin.Api/Controller/AdminLocalizationController.cs:
```diff
@@ -0,0 +1,40 @@
+    [ApiController]
+    [Produces("application/json")]
+    [Consumes("application/json")]
+    [Route("admin/content")]
```
add [Authorize]

idormenco commented on Src/DeUrgenta.Admin.Api/Controller/AdminLocalizationController.cs:
```diff
@@ -0,0 +1,40 @@
+            _i18NProvider = i18NProvider;
+        }
+
+        [HttpGet]
```
can you please add SwaggerResponseExample and SwaggerResponse tags on all newly created endpoints

idormenco commented on Src/DeUrgenta.Content.Api/Controller/ContentController.cs:
```diff
@@ -0,0 +1,43 @@
+
+namespace DeUrgenta.Content.Api.Controller
+{
+    public class ContentController : ControllerBase
```
add annotations 
```
    [ApiController]
    [Route("backpack")]
    [Produces("application/json")]
    [Consumes("application/json")]
    [Authorize]
```

DSVDavid committed:
Controllers updated
- added authorization and missing attributes
- added Swagger Documentation for endpoints

DSVDavid committed:
Merge branch 'feature/i18n_content' of https://github.com/DSVDavid/de-urgenta-backend into feature/i18n_content

DSVDavid committed:
Merge branch 'code4romania:develop' into feature/i18n_content

idormenco reviewed (approved):
looks good
